starter-code/index.html

```diff
@@ -3,40 +3,99 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="../starter-code/css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+<header class="headerBackground">
+
+<div class="img">
+  <img src="../starter-code/images/spotify-logo.png" width="400px" alt="spotify-logo"></div>
```

Comment:
May be you can try with a smaller width for your logo.
Also try not to use inline css like this, it's better to keep all you styles on a dedicated file :)
And it will be easier for you to change it !

This is the only point I have to mention, otherwise good job :)